### PR TITLE
perf: level of some output

### PR DESCRIPTION
**Pull request by RockChinQ**
While using this lib to create other projects that using `logging` module, some log with `INFO` level outputs by this lib may mess up the console and log file.

### CODE REVIEW

File: src/revChatGPT/V1.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def ask(
             try:
                 line = json.loads(line)
             except json.decoder.JSONDecodeError:
-                log.info("Error parsing JSON")
+                log.error("Error parsing JSON")
```
Comment (acheong08) [inline src/revChatGPT/V1.py:267]: This isn't a true error. Happens often due to bad escapes by OpenAI. Has to be info

**Review by acheong08 — CHANGES_REQUESTED** [review]

File: src/revChatGPT/V1.py (hunk 1 above)
Comment (acheong08) [reply src/revChatGPT/V1.py:267]: And can just be passed since the next line would have better escapes

Comment (RockChinQ) [reply src/revChatGPT/V1.py:267]: I see, so this should also be `debug`  or be hiden for upper caller?

Comment (acheong08) [reply src/revChatGPT/V1.py:267]: I suppose just remove logging for that in general. It doesn't do much

**Comment by RockChinQ** [comment]
This line has been removed now.

**Review by acheong08 — APPROVED** [review]

**Pull request merged**